ros_github_scripts/ci_for_pr.py

```diff
@@ -317,10 +317,10 @@ def comment_results(
         for pull in pulls:
             logger.info(f'Posting info as comment on {pull.html_url}')
             pull.create_issue_comment(body=contents)
-        logger.info('>>> AUTO-COMMENTED BELOW CONTENT ON ALL PRS <<<')
+        print('\n>>> AUTO-COMMENTED BELOW CONTENT ON ALL PRS <<<')
     else:
-        logger.info('>>>> COPY-PASTE BELOW CONTENT TO PRS <<<')
-    logger.info(contents)
+        print('\n>>> COPY-PASTE BELOW CONTENT TO PRS <<<')
+    print(contents)
```

Comment:
i believe this contents should be printed as standard output for user, regardless of logger level.